VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.html

```diff
@@ -1,140 +1,136 @@
-      colour="grey"
-      (click)="toggleVideoMute()">
-      <span class="far" [class]="videoMuted ? 'fa-video-slash' : 'fa-video'"></span>
-    </div>
-    
-    <div class="icon-button"
-      id="toggle-audio-mute-img" 
-      appTooltip
-      *ngIf="!remoteMuted || isJudge"
-      [text]="audioMuted ? 'Unmute microphone' : 'Mute microphone'"
-      colour="grey"
-      (click)="toggleMute()">
-      <span class="far" [class]="audioMuted ? 'fa-microphone-slash' : 'fa-microphone'"></span>
-    </div>
-    
-    <div class="icon-button"
-      id="toggle-audio-mute-locked-img" 
-      appTooltip
-      *ngIf="remoteMuted && !isJudge"
-      text="You have been been remote muted"
-      colour="grey">
-      <span class="far fa-microphone-slash red"></span>
-    </div>
-    
-    <div class="icon-button"
-      id="toggle-hand-raised-img" 
-      appTooltip
-      *ngIf="!isPrivateConsultation && !isJudge"
-      [text]="handRaised ? 'Lower hand' : 'Raise hand'"
-      colour="grey"
-      (click)="toggleHandRaised()">
-      <span class="far fa-hand-paper" [class]="{ 'yellow': handRaised }"></span>
-    </div>
+        <div class="icon-button"
+             id="toggle-chat-panel"
+             appTooltip
+             *ngIf="isJudge && !isPrivateConsultation"
+             text="{{ 'private-consultation-room-controls.chat-panel' | translate }}"
+             colour="grey"
+             (click)="togglePanelStatus('Chat')">
+            <span class="far fa-comment"></span>
+            <span class="badge" *ngIf="unreadMessageCount > 0">{{unreadMessageCount > 9 ? '9+' : unreadMessageCount }}</span>
+        </div>
+
+        <img *ngIf="!screenShareStream && canShowScreenShareButton"
+             appTooltip
+             text="{{ 'private-consultation-room-controls.share-my-screen' | translate }}"
+             colour="grey"
+             id="start-screenshare-img"
+             class="icon-button"
+             src="/assets/images/share-screen.png"
+             alt="{{ 'private-consultation-room-controls.start-sharing-screen' | translate }}"
+             (click)="startScreenShare()" />
+        <img *ngIf="screenShareStream"
+             appTooltip
+             text="{{ 'private-consultation-room-controls.stop-sharing-my-screen' | translate }}"
+             colour="red"
+             id="stop-screenshare-img"
+             class="icon-button"
+             src="/assets/images/stop-share.png"
+             alt="{{ 'private-consultation-room-controls.stop-sharing-screen' | translate }}"
+             (click)="stopScreenShare()" />
+
+        <div class="flexBar"></div>
+
+        <div class="icon-button"
+             id="toggle-self-view-img"
+             appTooltip
+             *ngIf="!audioOnly"
+             text="selfViewOpen ? '{{ 'private-consultation-room-controls.selfview-toggle-true' | translate }}' : '{{ 'private-consultation-room-controls.selfview-toggle-false' | translate }}'"
```

Comment:
this files got a lot of `'` inside `'` that might not work?

Reply:
Looks good for now